app_dart/lib/src/service/buildbucket.dart

```diff
@@ -70,6 +70,7 @@ class BuildBucketClient {
     }
 
     httpRequest.write(json.encode(request.toJson()));
+    print(json.encode(request.toJson());
```

Comment:
Print a description of whats being output here?

Reply:
Would print show in the remote logs?

Reply:
Removed, this was from a mock checkout

Prints do show in stackdriver, but they're not properly correlated (it's very useful if you don't want to plumb logging instances to somewhere like here)

Reply:
Good to know, thanks!